docs/specs/user-model.md

```diff
@@ -306,6 +308,25 @@ The mode of secondary authentication is configurable. They are as follows:
 
 The default mode is `if_exists`.
 
+```yaml
+authentication:
+  secondary_authentication_mode: (disabled|required|if_exists)
+```
+
+#### Secondary Authenticator Create Policy
+
+The secondary authenticator creation policy specifies how the secondary authenticators are set up during signup.
+
+- `any`: When `secondary_authentication_mode` is `required`, users have to set up one authenticator during signup. If multiple secondary authenticators are enabled in the project, users can choose one of them from the list.
+- `all`: When `secondary_authentication_mode` is `required` or `if_exists`, users need to set up all the enabled secondary authenticators based on the configuration order during signup.
```

Comment:
> When `secondary_authentication_mode` is `required` or `if_exists`

When the mode is `if_exists`, I think we should not force the user to set up any MFA during signup.